services/system/Transact/src/RTransact.cpp

```diff
@@ -122,6 +122,17 @@ namespace
       return to<Transact>().checkFirstAuth(id, *trx.transaction);
    }
 
+   bool isBooted()
+   {
+      auto status =
+          Transact::Tables(Transact::service).open<TransactStatusTable>().get(std::tuple{});
+      if (!status)
+         return false;
+      if (!status->bootTransactions)
+         return false;
```

Comment:
Should return true. Also, I suspect that it's more reliable to check `status->enforceAuth`.